packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -1,12 +1,152 @@
+import { M } from '@endo/patterns';
+import { makeError, q } from '@endo/errors';
+
+import { CctpTxEvidenceShape, StatusManagerEntryShape } from '../typeGuards.js';
+import { TxStatus } from '../constants.js';
+
 /**
  * @import {Zone} from '@agoric/zone';
+ * @import {CctpTxEvidence, NobleAddress, StatusManagerEntry} from '../types.js';
  */
 
+/** @param {CctpTxEvidence} evidence */
+export const getStatusKey = evidence => {
+  const {
+    tx: { amount, forwardingAddress },
+  } = evidence;
```

Comment:
more concise, and, IMO, just as clear:

```suggestion
  const { amount, forwardingAddress } = evidence.tx;
```

Reply:
agree, updated